features/rhelah/subman_ok.feature

```diff
@@ -16,7 +16,7 @@ Background: Atomic hosts are discovered
 
   Scenario: 2. subscription-manager plugin attaches existing pools
        Given cloud-init on "all" host is running
-       Then check if it successfully attached defined pools
+       Then check if it successfully attached existing pools
```

Comment:
This is still too generic, in my opinion.  Please update this line and the associated step here:

https://github.com/aweiteka/UATFramework/pull/43/files#diff-6b8f0fd24b0008acc91407983ef6f088R102